refactor/results/app/api/api_v0/endpoints/workflow.py

```diff
@@ -274,16 +278,32 @@ def update_result(*, dispatch_id: str, task: bytes = File(...)) -> Any:
         return {"response": "Task updated successfully"}
 
 
-@router.delete("/results/{dispatch_id}", status_code=200)
-def delete_result(*, dispatch_id: str) -> Any:
-    # Retrieve file path from db and request deletion from data service
-    filename = _get_result_from_db(dispatch_id, "filename")
-    r = requests.delete(f"http://{base_url}/delete", params={"obj_name": filename})
+@router.delete("/results", status_code=200, response_model=DeleteResultResponse)
+def delete_result(*, dispatch_ids: List[str] = Query([])) -> DeleteResultResponse:
```

Comment:
`List[str]` will expect the query to be of the form: `?dispatch_ids=xxx&dispatch_ids=xxx&dispatch_ids=xxx` which is verbose and redundant but it works. A comma-delimited list seems better here but it seems to require a customer query parser so probably not worth the effort.